app/templates/briefs/view_response_result.html

```diff
@@ -0,0 +1,52 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Your response to ‘{{ brief.title }}’ - Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <header class="page-heading-smaller">
```

Comment:
Is there a reason why we're not using the toolkit template for this http://alphagov.github.io/digitalmarketplace-frontend-toolkit/page-heading.html?